tools/serverpod_cli/lib/src/analyzer/models/validation/restrictions/default.dart

```diff
@@ -120,4 +122,33 @@ class DefaultValueRestriction extends ValueRestriction {
     }
     return errors;
   }
+
+  List<SourceSpanSeverityException> _doubleValidation(
+    dynamic value,
+    SourceSpan? span,
+  ) {
+    if (value is double) return [];
+
+    var errors = <SourceSpanSeverityException>[];
+
+    if (value is! String) {
+      errors.add(
+        SourceSpanSeverityException(
+          'The "$key" value must be a valid double.',
+          span,
+        ),
+      );
+    }
+
+    double? parsedValue = double.tryParse(value);
+    if (parsedValue == null) {
+      errors.add(
+        SourceSpanSeverityException(
+          'The "$key" value must be a valid double.',
```

Comment:
Suggestion: Same suggestion for improvement here as for the int parsing, add an example.